src/components/app-header/components/alarms-control/alarms-control.tsx

```diff
@@ -17,13 +16,16 @@ export const AlarmsControl = ({
       {criticalAlarmsCount > 0 && <AlarmIndicator alarmType="critical" />}
       {warningAlarmsCount > 0 && <AlarmIndicator alarmType="warning" />}
     </IndicatorsContainer>
-    <StyledButtonContainer
-      href="#"
+
+    <Button
+      flavour="borderless"
+      neutral
+      themeType="dark"
```

Comment:
What is themeType?

Reply:
Please refer to the [discussion](https://github.com/netdata/netdata-ui/pull/110#discussion_r479283022) having about this on the ui kit. If it's not clear we can discuss more.

Reply:
So, now the Button component should be aware of what themes exactly do exist, to correctly present itself?

Reply:
Nope, that's just a way to provide custom styles (in that case styles from a different theme) without the need to write the styles yourself.